### ISSUE TITLE
Easy installtion manual

### ISSUE THREAD

**Issue by jayabie** [Phuks-co/throat#516]
Hello Team,

I'm a beginner and i would like to install Thraot on a VPS.
is there any easy manual i can follow will allow me to make a full setup?

**Comment by jazzakek** [comment]
Yes, the setup guide can be found here: https://github.com/Phuks-co/throat/blob/master/README.md
